src/Lumper.Lib/Jobs/StripperJob.Block.cs

```diff
@@ -68,20 +62,22 @@ protected static bool MatchKeyValue(Prop filterProp, Entity.EntityProperty entit
             if (filterKey != entityProp.Key)
                 return false;
 
-            if (filterValue.Length > 2 && filterValue.StartsWith('/') && filterValue.EndsWith('/'))
+            if (filterValue.Length > 2 && filterValue.StartsWith('/') && filterValue.EndsWith('/') &&
+                entityProp.ValueString is not null)
             {
-
-                //todo perl regex or warning
-                var regex = new Regex(
-                    filterProp.Value[1..(filterProp.Value.Length - 2)]);
-
-                var ret = regex.IsMatch(entityProp.ValueString);
-                return ret;
+                try
+                {
+                    var regex = new Regex(filterValue[1..^2]);
+                    return regex.IsMatch(entityProp.ValueString);
+                }
+                catch (Exception _) when (_ is ArgumentException or ArgumentNullException)
+                {
+                    Logger.Warn(
+                        $"Error: Invalid regex {filterValue}. Stripper uses Perl-style regexes, Lumper uses .NET; your regex may need adjusting.");
```

Comment:
Yes this will just catch cases where the pattern is invalid or produces `null` (not sure when that is but the `Regex` ctor is annotated with throwing that.

According to annotations the one other error this block could throw would be a time-out exception, when a backtracking regex blows up. Reminds me that I should maybe add some safety checks to the ReplaceTextureJob regex.

I could get rid of the explicit exceptions here but I like specifying exactly what we think might actually throw. Jobs are allowed to throw, they're just exit immediately, that's probably the behaviour we want for any other exceptions. For these two, we can just warn and keep running the job.